# Request
int32 pump_id
int32 ppm_speed
int32 duration
---
# No response